templates/traas.yaml

```diff
@@ -95,15 +100,40 @@ resources:
       type: string
       value: |
         #!/bin/bash
+        # Ensure dev requirements for pre-provisioned nodes
+        setenforce 0
+        sed -i 's/^SELINUX=.*/SELINUX=permissive/' /etc/selinux/config
```

Comment:
it should be the case for tripleo-ci, right. This is a leftover from my fork, I deploy off road there, not using CI scripts - only quickstart roles on vanilla nodes...